lib/workers/repository/__fixtures__/master-issue_with_8_PR.txt

```diff
@@ -6,6 +6,7 @@ These branches will be created by Renovate only once you click their checkbox be
 
  - [ ] <!-- approve-branch=branchName1 -->pr1
  - [ ] <!-- approve-branch=branchName2 -->pr2
+ - [ ] <!-- approve-all-pending-prs -->🔐 **Create all pending approval PRs at once** 🔐
```

Comment:
We should use _one_ word for the same concept. Right now we use both `create` and `open`.

- `Create all ... PRs at once`
- `Open all ... PRs at once`

Reply:
I was thinking to use both "open" and "create" since at the "rate-limited" section, the concept is that the PRs are already open but are just blocked by rate limitation, and at the "pending approval" section it was not yet created since it requires approval first.
If you think it is better to use same wording, I would go with "create" since we use the term creation in the titles of the two.

Reply:
Yeah that makes sense. I'll let you decide what you like best then. 😉